lnclient/lnd/lnd.go

```diff
@@ -719,16 +740,14 @@ func (svc *LNDService) GetOnchainBalance(ctx context.Context) (*lnclient.Onchain
 	}, nil
 }
 
+// TODO: SendCoins
```

Comment:
Note: I think this should be prioritized over getting the network graph / debug methods

Reply:
Yes this is not related to #246, thought of working on the lnd issue so I can add both at once so shifted to that, will add both now and open this for review 👍